mongodb/src/mongodb/Module.md

```diff
@@ -0,0 +1,99 @@
+Connects to Mongo DB from Ballerina.
+
+# Module Overview
+
+The Mongo DB connector allows you to connect to Mongo DB from Ballerina and perform `insert`, `find`, `findOne`, `replaceOne`, `delete` operations.
+
+## Compatibility
+
+|                             |       Version               |
+|:---------------------------:|:---------------------------:|
+| Ballerina Language          | 1.0.1                       |
+| Mongo DB                    | V4.2.0                      |
+
+## Sample
+
+First, import the `wso2/mongodb` module into the ballerina project.
+
+```ballerina
+import ballerina/config;
+import ballerina/log;
+import wso2/mongodb;
+
+public function main() returns error?
+{
```

Comment:
It seems mismatch of open and close curly braces. Also, formatting is not correct.

Reply:
Fixed